homeassistant/components/home_connect/quality_scale.yaml

```diff
@@ -2,57 +2,42 @@ rules:
   # Bronze
   action-setup: done
   appropriate-polling:
-    status: todo
+    status: done
     comment: |
-      The integration polls the Home Connect API every minute. This frequency can be lowered
-      because we receive push updates from the API so we don't need to poll that often.
+      Polling is performed only once at the configuration entry setup.
```

Comment:
I think we'll need to revisit this to handle rate limits for users that are using more than a couple of appliances.

Reply:
```suggestion
      Polling is performed only once at the configuration entry setup but needs to be optimized further to avoid rate limits.
```